src/main/java/chess/dao/GameDaoImpl.java

```diff
@@ -0,0 +1,117 @@
+package chess.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import chess.Game;
+
+public class GameDaoImpl implements GameDao {
+
+    private static final String URL = "jdbc:mysql://localhost:13306/chess";
+    private static final String USER = "user";
+    private static final String PASSWORD = "password";
+
+    private final Game game;
+
+    public GameDaoImpl(Game game) {
+        this.game = game;
+    }
+
+    @Override
+    public Connection getConnection() {
+        Connection connection = null;
+        try {
+            connection = DriverManager.getConnection(URL, USER, PASSWORD);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return connection;
```

Comment:
커넥션코드가 응집되지 않는 것 같아요. 재활용할 수 있는 방법이 없을까요?



---

```suggestion
        try {
           return  DriverManager.getConnection(URL, USER, PASSWORD);
        } catch (SQLException e) {
          throw....
        }
```
또 이런식으로하면 불필요한 null할당은 피할 수 있을 것 같아요

Reply:
위의 내용처럼 변경하고 객체를 분리해보았습니다. 감사합니다!!!